lib/logic/control.dart

```diff
@@ -41,7 +41,6 @@ abstract class Control {
   /// Must be called when the control needs to be initialized.
   @mustCallSuper
   void init() {
-    assert(_disposed.value);
```

Comment:
Just removing this is incorrect, this is asserting that controls are not initialized twice. This is not ideal however, and we should rather silently ignore additional `init` calls in subclasses, by checking the `disposed`.

I'm not sure why you changed the rest of the code in this pull request, can you clarify?

Reply:
@nt4f04uNd Is this ok with you, or do you want another solution?

Reply:
It is OK